gulpfile.js

```diff
@@ -40,3 +45,48 @@ gulp.task('coverage', function () {
     });
     open('coverage/lcov-report/index.html');
 });
+
+gulp.task('create-zip', function () {
+    var hash;
+    var status = child_process.execSync('git status -uno -s').toString().trim();
+    if (!/^\s*$/.test(status)) {
+        if (!argv.force) {
+            console.log('Refusing to create a release for a modified branch. Pass the --force flag if you know what you\'re doing.');
+            return;
+        }
+        hash = 'local-modifications';
+    } else {
+        hash = child_process.execSync('git rev-parse HEAD').toString().trim();
+    }
+
+    var zipName = 'cesium-concierge-' + hash + '.zip';
+
+    var serverFiles = gulp.src([
+            'lib/**',
+            'index.js'],
+        {
+            base: '.'
+        });
+
+    //We don't run post install in production.
+    packageJson.version = packageJson.version + '-' + hash + '.0';
```

Comment:
We can skip this part, which means we can get rid of the `build` directory completely.  Just delete `npm-shrinkwrap.json` at the end instead of moving it into `build`